cli/command/swarm/progress/root_rotation.go

```diff
@@ -0,0 +1,171 @@
+package progress
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/progress"
+	"github.com/docker/docker/pkg/streamformatter"
+	digest "github.com/opencontainers/go-digest"
+)
+
+const (
+	certsRotatedStr = "  nodes rotated TLS certificates"
+	rootsRotatedStr = "   nodes rotated CA certificates"
```

Comment:
Right now the output looks like this:

```
desired root digest: sha256:a594409c21f2339d4fa5a207549e500574d6a9872dcd040f4e63e461aa969302                                                                                      
  nodes rotated TLS certificates: 1/1 done [==================================================>]                                                                                  
   nodes rotated CA certificates: 1/1 done [==================================================>] 
```

I have a slight preference for:

```
desired root digest: sha256:a594409c21f2339d4fa5a207549e500574d6a9872dcd040f4e63e461aa969302                                                                                      
  nodes rotated TLS certificates: 1/1 done [==================================================>]                                                                                  
  nodes rotated CA certificates:  1/1 done [==================================================>] 
```

But this is not important